tests/test_projectedLikelihood.py

```diff
@@ -212,14 +221,15 @@ def testDirect(self):
         self.checkLikelihood(l0b, data)
         # test with constant weights, using both ctors
         ctrl.usePixelWeights = False
+        # for the usePixelWeights = False relax checkLikeihood to pass on osx-arm64
```

Comment:
I would add a comment here that this relaxation is required because this particular code path uses single-precision exp/log/sum within the c++ code and this can lead to larger architecture differences.

Reply:
Added